tests/test_decorators.py

```diff
@@ -36,10 +42,10 @@ def f ():
         self.assertEqual(f(), 42)
 
     def test_timeit2 (self):
-        log = BytesIO()
+        log = StringIO()
         @linkcheck.decorators.timed(log=log, limit=0)
         def f ():
             time.sleep(1)
             return 42
         self.assertEqual(f(), 42)
-        self.assertTrue(log.getvalue())
+        self.assertEqual(log.getvalue(), 'f took 1.00 seconds\n()\n{}\n')
```

Comment:
This looks right to me: the default value of `log` is `sys.stderr`, which is definitely a text stream and not a byte stream on Python 3.

(The decorator itself looks like debug code to be inserted while developing and currently has no uses outside of the test suite, so getting it wrong won't cause any harm to users.)